PocketKit/Sources/PocketKit/Main/RegularSavesCoordinator.swift

```diff
@@ -178,6 +186,15 @@ extension RegularSavesCoordinator {
         viewController.popoverPresentationController?.sourceView = sortFilterVM.sender as? UIView
         self.viewController.present(viewController, animated: true)
     }
+
+    private func updateSearchScope(isFromSaves: Bool) {
+        savesViewController.navigationItem.searchController?.isActive = true
+        if isFromSaves {
+            savesViewController.navigationItem.searchController?.searchBar.selectedScopeButtonIndex = 0
+        } else {
+            savesViewController.navigationItem.searchController?.searchBar.selectedScopeButtonIndex = 1
+        }
+    }
```

Comment:
since this utilizing the `isFromSaves` and `savesViewController`, what do we think about adding this logic to the view controller? This way we won't need to `isFromSaves` and can call `savesViewController.updateScope()`. 

We are also doing similar logic so maybe we can consolidate it to one place

Reply:
sounds good! If i understood correctly, the changes will be in the next commit

Reply:
Ah I was thinking that we forgo the `isFromSaves` entirely from the Coordinator since the `savesViewController` should know what's selected, maybe something like this? Let me know if you think this is better for a more maintainable codebase

```
func updateSearchScope() {
        let isFromSaves = selectedView == SelectedView.saves
        if isFromSaves {
            navigationItem.searchController?.searchBar.selectedScopeButtonIndex = 0
        } else {
            navigationItem.searchController?.searchBar.selectedScopeButtonIndex = 1
        }
    }
```

also, lets make sure to make the same change to the Compact coordinator